From 44722a4ec5f54803f092d81b4c08e055052aa138 Mon Sep 17 00:00:00 2001
From: Wolfgang Bauer <wbauer@tmo.at>
Date: Thu, 17 Aug 2017 16:13:31 +0200
Subject: [PATCH] Use qdbus-qt5 and qdbusviewer-qt5

The Qt4 version might not be installed any more, and it's needed for
D-BUS actions like spectacle's "Print Screen" shortcuts.
---
 kcm_hotkeys/actions/dbus_action_widget.cpp | 2 +-
 libkhotkeysprivate/actions/dbus_action.cpp | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

Index: khotkeys-5.15.80git.20200423T192354~9b4759c/kcm_hotkeys/actions/dbus_action_widget.cpp
===================================================================
--- khotkeys-5.15.80git.20200423T192354~9b4759c.orig/kcm_hotkeys/actions/dbus_action_widget.cpp	2020-04-24 08:10:26.252530468 +0200
+++ khotkeys-5.15.80git.20200423T192354~9b4759c/kcm_hotkeys/actions/dbus_action_widget.cpp	2020-04-24 08:11:06.703582629 +0200
@@ -117,7 +117,7 @@
 
 void DbusActionWidget::launchDbusBrowser() const
     {
-        auto *job = new KIO::CommandLauncherJob("qdbusviewer");
+        auto *job = new KIO::CommandLauncherJob("qdbusviewer-qt5");
         job->setUiDelegate(new KDialogJobUiDelegate(KJobUiDelegate::AutoHandlingEnabled, window()));
         job->start();
     }
Index: khotkeys-5.15.80git.20200423T192354~9b4759c/libkhotkeysprivate/actions/dbus_action.cpp
===================================================================
--- khotkeys-5.15.80git.20200423T192354~9b4759c.orig/libkhotkeysprivate/actions/dbus_action.cpp	2020-04-24 08:08:41.193493337 +0200
+++ khotkeys-5.15.80git.20200423T192354~9b4759c/libkhotkeysprivate/actions/dbus_action.cpp	2020-04-24 08:10:26.252530468 +0200
@@ -133,7 +133,7 @@
         }
     qDebug() << "D-Bus call:" << _application << ":" << _object << ":" << _function << ":" << args_list;
     KProcess proc;
-    proc << "qdbus" << _application << _object << _function << args_list;
+    proc << "qdbus-qt5" << _application << _object << _function << args_list;
     proc.startDetached();
     }
 
